src/script/ismine.h

```diff
@@ -28,7 +28,9 @@ enum isminetype
 /** used for bitflags of isminetype */
 typedef uint8_t isminefilter;
 
-isminetype IsMine(const CKeyStore& keystore, const CScript& scriptPubKey);
-isminetype IsMine(const CKeyStore& keystore, const CTxDestination& dest);
+isminetype IsMine(const CKeyStore& keystore, const CScript& scriptPubKey, bool& isInvalid, SigVersion = SIGVERSION_BASE);
```

Comment:
Could we get an explanation of the different arguments here? `isInvalid` seems to have a particular meaning and use that is non-obvious, so it would benefit reviewers.